internal/Yesod/Session/Manager.hs

```diff
@@ -18,7 +18,7 @@ data SessionManager tx m = SessionManager
   , storage :: forall a. StorageOperation a -> tx a
   -- ^ The storage backend
   , options :: Options tx m
-  , runTransaction :: forall a. tx a -> m a
+  , runDB :: forall a. tx a -> m a
```

Comment:
Why change this name? It seemed important to distinguish this as `tx a -> m a`

Reply:
I am sort of ambivalent on this - `runTransaction` is an iffy name when now applied to a storage backend that isn't actually transactional.

Reply:
For memcache, it's nothing at all (`id`).

For the SQL version, it was "get a connection from the pool to run the `ReaderT SqlBackend`" (`runDB`).

And, this is nothing but a curiosity, but for the mock in tests it's STM `atomically`.